src/python/pants/backend/docker/subsystems/docker_options.py

```diff
@@ -86,16 +92,33 @@ def register_options(cls, register):
             default=[],
             advanced=True,
             help=(
-                "Environment variables to set for `docker` invocations. "
+                "Environment variables to set for `docker` invocations.\n\n"
                 "Entries are either strings in the form `ENV_VAR=value` to set an explicit value; "
                 "or just `ENV_VAR` to copy the value from Pants's own environment."
             ),
         )
 
+        register(
+            "--run-args",
+            type=list,
+            member_type=str,
```

Comment:
Should use `shell_str` so that you don't want to worry about shlexing on the CLI

Reply:
> Should use `shell_str` 

Is this true also for the build args and env vars options? (My feeling say, yes)